app/views/staff/shared/xml/_activity.xml.haml

```diff
@@ -39,12 +39,10 @@
     %website= I18n.t("contact_info.website")
     %mailing-address
       %narrative= I18n.t("contact_info.mailing_address")
-  - if activity.recipient_country?
```

Comment:
I think my original interpretation of these "additive" steps to implementing `benefitting_countries` was to use the info *if* we have it, but if `benefitting_countries` aren't present, default to showing the legacy `recipient_*` fields until we officially remove them.

This is intended to cope with the intermediate phase when we haven't backfilled the values for the new field.

I'd have done this by using `benefitting_countries` if present, and `elsif`ing to use `recipient_country` or `recipient_region`, rather than deleting them.

Reply:
Yeah, good call. Makes sense to do all this piecemeal. Sorted now!